### PR TITLE
Mint form

### PR DESCRIPTION
**Pull request by Mussuranae**

### CODE REVIEW

File: apps/dashboard/src/app/mint/mint.component.ts [hunk 1]
```diff
@@ -1,9 +1,43 @@
...
+    jwPlayerId: new FormControl(''),
+    description: new FormControl(''),
+    metadata: new FormGroup({
+      runningTime: new FormControl(null, Validators.required),
+      countries: new FormArray([])
+    })
+  });
+
+  onSubmit() {
+    console.log(this.mintForm);
```
Comment (phpgeekfr) [inline apps/dashboard/src/app/mint/mint.component.ts]: work in progress?

File: apps/dashboard/src/app/mint/mint.component.ts [hunk 2]
```diff
@@ -1,9 +1,43 @@
...
+
+  get countries() {
+    return this.mintForm.controls.metadata.get('countries') as FormArray;
+  }
+
+  // CHIPS LOGIC
+  add(event: MatChipInputEvent) {
+    this.countries.push(new FormControl(event.value));
+    event.value = '';
+    event.input.value = '';
```
Comment (phpgeekfr) [inline apps/dashboard/src/app/mint/mint.component.ts]: you need to reset both ?

**Review by phpgeekfr — APPROVED** [review]

File: apps/dashboard/src/app/mint/mint.component.ts (hunk 1 above)
Comment (Mussuranae) [reply apps/dashboard/src/app/mint/mint.component.ts]: Yep, I don't do anything after that step

File: apps/dashboard/src/app/mint/mint.component.ts (hunk 2 above)
Comment (Mussuranae) [reply apps/dashboard/src/app/mint/mint.component.ts]: No the first one isn't mandatory. I remove it.

**Pull request merged**